frontend/src/lib/components/Editor.svelte

```diff
@@ -23,9 +19,13 @@
 	export let formatAction: (() => void) | undefined = undefined
 	export let automaticLayout = true
 	export let websocketAlive = { pyright: false, black: false, deno: false }
+	export let extraLib: string = buildExtraLib()
```

Comment:
this is only useful if the lang is deno nd the editor is also for json and python, we can skip that if not right lang

Reply:
This logic should be extracted from the Editor component. Will do in an other PR.